yearn/prices/curve.py

```diff
@@ -60,7 +60,9 @@
         'voting_escrow': '0x5f3b5DfEb7B28CDbD7FAba78963EE202a494e2A2',
         'gauge_controller': '0x2F50D538606Fa9EDD2B11E2446BEb18C9D5846bB',
         # additional factories not listed as metapool factory
-        'factories': ['0xF18056Bbd320E96A48e3Fbf8bC061322531aac99'],
```

Comment:
maybe remove altogether, we can always add later and there is always a reference in history

Reply:
done